src/Modules/Manage/Dnn.PersonaBar.Users.Tests/CommandTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Dnn.PersonaBar.Library.Prompt;
+using DotNetNuke.Entities.Portals;
+using Dnn.PersonaBar.Library.Prompt.Models;
+using DotNetNuke.Entities.Users;
+
+namespace Dnn.PersonaBar.Users.Tests
+{
+    public abstract class CommandTests
+    {
+        protected ConsoleCommandBase command;
+        protected PortalSettings portalSettings;
+        protected ConsoleErrorResultModel errorResultModel;
+
+        protected int testPortalId = 0;
+
+        protected abstract ConsoleCommandBase DependencyInitializer();
```

Comment:
That is a funky name. Would `CreateCommand` or `GetCommand` work? 
Or even `protected abstract ConsoleCommandBase Commend { get; }`